qe_metrics/libs/qe_metrics_base.py

```diff
@@ -0,0 +1,19 @@
+from abc import ABC
+
+from pyaml_env import parse_config
+from simple_logger.logger import get_logger
+
+
+class QeMetricsBase(ABC):
+    def __init__(self, creds_file: str) -> None:
+        """
+        Initialize the QeMetricsBase class
```

Comment:
small comments, please add empty line after description in docstring (before args).
Otherwise looks good.